src/o365/yammer/commands/user/user-list.ts

```diff
@@ -0,0 +1,195 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import YammerCommand from "../../../base/YammerCommand";
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  letter?: string;
+  sortBy?: string;
+  reverse?: boolean;
+  limit?: number;
+  groupId?: number;
+}
+
+class YammerUserListCommand extends YammerCommand {
+  protected items: any[];
```

Comment:
This can be `private` since typically one command does not inherit from another. I believe originally this comes from an abstract command class for getting all items which is why it was protected.

Reply:
I am getting a 'private' modifier cannot appear on a module or namespace element.ts(1044) by doing so. Keeping it as it is for now

Reply:
Odd, it's transpiling here just fine 😕